ArticleTemplates/assets/js/modules/ophan-iframe.js

```diff
@@ -0,0 +1,20 @@
+const handleIframeMessage = (messageEvent) => {
+    const correctType = messageEvent.data.type === 'ophan-iframe-click-event';
+    const correctOrigin = messageEvent.origin === 'https://www.theguardian.com' ||
+        messageEvent.origin === 'https://m.code.dev-theguardian.com/';
```

Comment:
Does this mean all `iframe` sources will be on the dotcom domain? As in, all relevant `iframe`s on the page will look something like:

```html
<iframe src="https://www.theguardian.com/..." />
```

Reply:
Yes, I want to make sure we don't accept any message from any origin (thinking about twitter embed and such)

Reply:
So all newsletter embeds on apps are served from the dotcom domain?